spec/features/users_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'should get all the users of the app' do
+      get '/users'
+      expect(response).to have_http_status(200)
+    end
+
+    it 'should render the index page with every existing user' do
+      get '/users'
+      expect(response).to render_template('index')
+    end
+
+    it 'should render the index page with correct text in the body' do
+      get '/users'
+      expect(response.body).to include('ALL OUR')
+    end
+
+    it 'should render the show page with specifics of one user' do
+      get '/users/1'
+      expect(response).to have_http_status(200)
+    end
+
+    it 'should render the show page with specifics of one user' do
+      get '/users/1'
+      expect(response).to render_template('show')
+    end
+
+    it 'should render the show page with specifics of one user' do
+      get '/users/1'
+      expect(response.body).to include('Number of posts')
+    end
+  end
+end
```

Comment:
- [ ] @Team I noted that you deleted the previous test files, and added a new test file however the new tests don't align with the required tests from the project requirement, kindly consider adding the required tests, it would be nice if you separated the tests for each page, eg `user_show_spec, user_index_spec`, below is a list of all tests required, kindly consider adding all the tests below, this will ensure you have implemented even the changes previously requested.
![image](https://github.com/AlaaAlsalem/blogaapp/assets/61408860/a3ea6251-7cd4-428f-8251-727c181eeb76)
![image](https://github.com/AlaaAlsalem/blogaapp/assets/61408860/f0a99100-4458-4fac-b1e5-aa44b2a4478e)

(Here is a [capybara cheat sheet](https://gist.github.com/zhengjia/428105)that will give you tips on how to write tests)